### PR TITLE
Fix LLM Factory not getting APIKeys

### PR DESCRIPTION
**Pull request by baldurrensch**

### CODE REVIEW

File: src/DependencyInjection/InstructriceExtension.php [hunk 1]
```diff
@@ -60,22 +60,24 @@ class InstructriceExtension extends ConfigurableExtension
...
-                DeepInfra::class => $mergedConfig['deepinfra']['api_key'] ?? null,
-                Fireworks::class => $mergedConfig['fireworks']['api_key'] ?? null,
-                Groq::class => $mergedConfig['groq']['api_key'] ?? null,
-                Mistral::class => $mergedConfig['mistral']['api_key'] ?? null,
-                Ollama::class => $mergedConfig['ollama']['api_key'] ?? null,
-                OpenAi::class => $mergedConfig['openai']['api_key'] ?? null,
-                Perplexity::class => $mergedConfig['perplexity']['api_key'] ?? null,
-                Together::class => $mergedConfig['together']['api_key'] ?? null,
-            ],
+            '$apiKeys' => $apiKeys,
```
Comment (adrienbrault) [inline src/DependencyInjection/InstructriceExtension.php:80]: This part can be removed actually!

File: src/DependencyInjection/InstructriceExtension.php [hunk 2]
```diff
@@ -91,6 +93,7 @@ protected function loadInternal(array $mergedConfig, ContainerBuilder $container
         $definition->setArguments([
             '$client' => new Reference(SymfonyStreamingClient::class),
             '$logger' => new Reference('logger'),
+            '$apiKeys' => $apiKeys,
```
Comment (adrienbrault) [inline src/DependencyInjection/InstructriceExtension.php:96]: So it was actually missing here

**Comment by adrienbrault** [comment]
Thank you!

File: src/DependencyInjection/InstructriceExtension.php (hunk 2 above)
Comment (baldurrensch) [reply src/DependencyInjection/InstructriceExtension.php:96]: Good point. Updated.

**Pull request merged**